src/main/java/seedu/address/model/food/Macronutrient.java

```diff
@@ -1,9 +1,10 @@
 package seedu.address.model.food;
 
+import static seedu.address.commons.util.AppUtil.checkArgument;
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
 public abstract class Macronutrient {
-    private String name;
+    private String type;
```

Comment:
I think we should rename this to `macronutrientType` or something so we don't confuse this `type` with a java type.

Reply:
Ok I will change it